src/drivers/sf1xx/sf1xx.cpp

```diff
@@ -75,18 +76,18 @@
 #include <board_config.h>
 
 /* Configuration Constants */
-#define SF10A_BUS 		PX4_I2C_BUS_EXPANSION
-#define SF10A_BASEADDR 	0x55
-#define SF10A_DEVICE_PATH	"/dev/sf10a"
+#define SF1XX_BUS 		PX4_I2C_BUS_EXPANSION
+#define SF1XX_BASEADDR 	0x55
+#define SF1XX_DEVICE_PATH	"/dev/sf1xx"
 
 /* Device limits */
-#define SF10A_MIN_DISTANCE 	(0.01f)
-#define SF10A_MAX_DISTANCE 	(25.0f)
+#define SF1XX_MIN_DISTANCE 	(0.01f)
+#define SF1XX_MAX_DISTANCE 	(120.0f)
 
```

Comment:
In SF11/c datasheet specify that sensor returns 130m in case of signal loss. In SF10/x there are no such section.Also in SF10/x there are no (officially) i2c command to identify device. So if someone has sf10/x lidar it can test this case. I'll add SENS_EN_SF1XX to select proper hw runtime if needed